i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/centreon-nsclient-tutorial.md

```diff
@@ -188,6 +188,15 @@ net start nscp
 
 ![image](../../../../assets/integrations/plugin-packs/how-to-guides/centreon-nsclient-tutorial-webserver.png)
 
+## Allowed Ciphers and pentesting tools 
+
+To secure your client-server communication and avoid alerts generation in security auditing tools like Nessus, please apply the following directive to 
+your nsclient.ini configuration: 
```

Comment:
LGTM. But are you sure that the requirements are meet in the default configuration file ?
(All the page seems translated in french)